frontend/src/pages/Roles/Roles.tsx

```diff
@@ -30,11 +37,15 @@ function Roles() {
                     <div className="h-[300px]">
                         <UserList allUsers={allUsers} setActiveRole={setActiveRole} />
                     </div>
+                    {/* Return button */}
+                    <div className="mt-10">
+                        <button className="text-3xl text-[#ffffff] bg-[#262b6c] p-4 border-4" onClick={returntoProfile} >Return to Profile</button>
```

Comment:
maybe add a "rounded-md" and change border to "border-2". Just for consistency across components

Reply:
done